content/events/2025-05-20-Jugend-hackt-Workshop-TurtleStitch

```diff
@@ -0,0 +1,10 @@
+---
+title: "Jugend hackt Lab: Münster | Workshop: Einführung in TurtleStitch"
+date: 2025-05-20
```

Comment:
@SandraMarie2025 das Datum sollte das sein, an welchem Tag es auf die Website gestellt wird. Da Datum jetzt noch in der Zukunft liegt wird das event noch nicht angezeigt